src/app/components/img-or-icon-selector/img-or-icon-selector.component.ts

```diff
@@ -40,6 +41,7 @@ export class ImgOrIconSelectorComponent implements AfterViewInit, OnChanges {
 
   ngOnChanges(changes: SimpleChanges): void {
     this.imgUrl = changes.imgUrl ? (changes.imgUrl.currentValue as string) : '';
+    this.isImage = this.imgUrl !== undefined || this.imgUrl !== '';
```

Comment:
Can you just use `!this.imgUrl` trick, then anything "falsey" such as `null` and `undefined` and `false` will be `false`

Reply:
Oh sorry, just cherry picked that commit and merged, will check that out

Reply:
@msavy @BastianGem That check does not work because we run into cast problems and I want to avoid the !! operator because I personally do not like that.
But your idea with falsey thing brought me to the idea to remove the isImage at all and only check the imgUrl.
I openend https://github.com/apiman/apiman-developer-portal/pull/86.
Its less verbose not sure what way you prefer